test/map.jl

```diff
@@ -83,79 +65,59 @@ module TestMap
         map!(f, dests[1], args[1]...)
         map!(f, dests[2], args[2]...)
         @test isequal(dests[2], NullableArray(dests[1], mask...))
-        # lifted map!
-        map!(g, dests[1], args[1]...)
-        map!(g, dests[2], args[2]...; lift=true)
-        @test isequal(dests[2], NullableArray(dests[1], mask...))
         # map
         map(f, args[1]...)
         map(f, args[2]...)
         @test isequal(dests[2], NullableArray(dests[1], mask...))
-        # lifted map
-        map(g, args[1]...)
-        map(g, args[2]...; lift=true)
-        @test isequal(dests[2], NullableArray(dests[1], mask...))
     end
 
     # test map over empty NullableArrays
     X = NullableArray(Int[])
     h1(x) = 5.0*x
-    h2(x::Nullable) = x
-    h2(x::Nullable...) = prod(x)
+    h2(x) = x
+    h2(x...) = prod(x)
 
     Z1 = map(h1, X)
     Z2 = map(h2, X)
-    if VERSION >= v"0.5.0-dev+3294"
-        @test isequal(Z1, NullableArray(Union{}[])) # no type inference
+    @test isempty(Z1)
+    if VERSION >= v"0.6.0-dev"
```

Comment:
please be specific with VERSION cutoffs like this, will this really work for all 0.6-dev versions?